packages/faustwp-cli/utils/isVerbose.ts

```diff
@@ -0,0 +1,9 @@
+export function isVerbose(): boolean {
+  const isVerboseEnvVar = process.env.FAUST_VERBOSE;
```

Comment:
Should we call it `FAUST_VERBOSE` or `FAUST_DEBUG`?

Reply:
+1 for `FAUST_DEBUG` to match WordPress' usage of `WP_DEBUG`

Reply:
I tend to agree. Will make the change!

Reply:
Updated 👍 Everything now references "debug" opposed to "verbose" now.